sensing/preprocessor/pointcloud/pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -48,269 +27,159 @@ catkin_package(
 include_directories(
   include
   ${Boost_INCLUDE_DIRS}
-  ${catkin_INCLUDE_DIRS}
-  ${EIGEN3_INCLUDE_DIRS})
+  ${PCL_INCLUDE_DIRS}
+  ${EIGEN3_INCLUDE_DIRS}
+  ${OpenCV_INCLUDE_DIRS})
 
-# Compare Map Filter
-add_library(
-  compare_map_filter
+
+# ========== Compare Map Filter ==========
+# -- Distance Based Compare Map Filter --
+ament_auto_add_library(distance_based_compare_map_filter SHARED
   src/compare_map_filter/distance_based_compare_map_filter_nodelet.cpp
+  src/filter.cpp
+  )
+target_link_libraries(distance_based_compare_map_filter ${Boost_LIBRARIES})
+rclcpp_components_register_node(distance_based_compare_map_filter
+  PLUGIN "pointcloud_preprocessor::DistanceBasedCompareMapFilterComponent"
+  EXECUTABLE distance_based_compare_map_filter_exe)
```

Comment:
Since this executable will be used as a replacement of original executable node, I think we should keep the name of the executable to `distance_based_compare_map_filter_node` instead of `distance_based_compare_map_filter_exe`. 
Same with other executables

Reply:
change `_exe` to `_node`